### ISSUE TITLE
README for examples/vault-consul-ami gives conflicting advice.

### ISSUE THREAD

**Issue by jemc** [hashicorp/terraform-aws-vault#167]
A comment in the CI script that pushes example AMIs says that TLS secrets should not be baked into a static AMI:
https://github.com/hashicorp/terraform-aws-vault/blob/6a091aab16d7be0a241a6bb07d59fc2f48baf412/_ci/publish-amis.sh#L42-L43

However, baking in TLS secrets into the static AMI is exactly [what the README tells you to do](https://github.com/hashicorp/terraform-aws-vault/blob/6a091aab16d7be0a241a6bb07d59fc2f48baf412/examples/vault-consul-ami/README.md#quick-start)

Can one or the other recommendation please be updated to not be in conflict with one another, so that readers have clear guidance?

**Comment by brikis98** [comment]
The comment in `publish-amis.sh` should warn about baking _unencrypted_ secrets into the AMI. Would you be up for a quick PR to fix that?

**Comment by jemc** [comment]
But, the README doesn't mention anything about a step where the secrets get encrypted, and the packer script seems to just copy the unencrypted files into the image: https://github.com/hashicorp/terraform-aws-vault/blob/6a091aab16d7be0a241a6bb07d59fc2f48baf412/examples/vault-consul-ami/vault-consul.json#L114-L119

**Comment by brikis98** [comment]
Yes, that's my point: there should be a warning about not baking unecrypted secrets into the AMI. There are many ways to encrypt data, so you could also consider sharing some of the options for that, such as using KMS.

**Comment by jemc** [comment]
I was hoping that someone with experience using this module could be the one to write this advice, not me. I came here looking for that advice and found it to be missing. Still trying to figure out what the common practice is for people who are actually using this module.
